avae/config.py

```diff
@@ -80,6 +80,7 @@
     "gaussian_blur": False,
     "normalise": False,
     "shift_min": False,
+    "tensorboard": True,
```

Comment:
There is a lot that is missing at the moment from README.md , I think it would be best to merge this branch and create an issue for updating the README.md to include the recent changes

Reply:
I agree that the README needs some major rewrite, but maybe for now we can add a line saying how to visualise the tensorboard? I'm running it and files are created but don't know what should I do next  😂